lyricsbot/domains/lyricsondemand/utils.py

```diff
@@ -0,0 +1,37 @@
+"""
+Helper.
```

Comment:
File for one helper or many? What they do?

Reply:
Done.